assembly.sbt

```diff
@@ -1,11 +1,4 @@
-// For sbt 0.13, requires plugin addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.11.2")
-
-import AssemblyKeys._ // put this at the top of the file
-
-assemblySettings
+mainClass in assembly := Some("tuffy.main.Main")
```

Comment:
sbt-assembly has changed how they do things, so this file can be pruned aggressively.  I'm not sure it's really required though, since we don't publish the assembly.